services/mirror/mirror.go

```diff
@@ -67,8 +71,8 @@ func Update(ctx context.Context, pullLimit, pushLimit int) error {
 			}
 			repo = m.Repo
 			item = SyncRequest{
-				Type:   PullMirrorType,
-				RepoID: m.RepoID,
+				Type:     PullMirrorType,
+				MirrorID: m.ID,
```

Comment:
`SyncPullMirror` needs to be called with a repository id and not the mirror id. So you would have to change the code there or rename `MirrorID` to `ReferenceID` or something.